src/components/Posts/ComposePost.tsx

```diff
@@ -96,7 +94,6 @@ const ComposePost = ({ placeholder }: ComposePostProps) => {
           </Box>
         </Box>
       </Box>
-      <Divider light />
```

Comment:
This divider helps differentiate the position of the ComposePost component and the PostItem component in the Timeline page. What was the reasoning for this change?